HARDHAT_README.md

```diff
@@ -0,0 +1,131 @@
+```
+
+| Option <img width=200/> | Example <img width=750/>| Description <img width=1000/> |
+|--------------|------------------------------------|--------------------------------|
+| testfiles  | `--testfiles "test/registry/*.ts"` | Test file(s) to run. (Globs must be enclosed by quotes.)|
+| solcoverjs | `--solcoverjs ./../.solcover.js` | Relative path from working directory to config. Useful for monorepo packages that share settings. (Path must be "./" prefixed) |
+| network    | `--network development` | Run with a ganache client over http using network settings defined in the Hardhat config. (Hardhat is the default network) |
+
+
+## Configuration
+
+Options can be specified in a `.solcover.js` config file located in the root directory of your project.
+
+**Config Example:**
+```javascript
+module.exports = {
+  skipFiles: ['Routers/EtherRouter.sol']
+};
+```
+
+| Option <img width=200/>| Type <img width=200/> | Default <img width=1300/> | Description <img width=800/> |
+| ------ | ---- | ------- | ----------- |
+| silent | *Boolean* | false | Suppress logging output |
+| client | *Object* | undefined | *Ganache only*: Useful if you need a specific ganache version. An example value is: `require('ganache-cli')` |
+| providerOptions | *Object* | `{ }` | *Ganache only*: [ganache-core options][1]  |
+| skipFiles | *Array* | `['Migrations.sol']` | Array of contracts or folders (with paths expressed relative to the `contracts` directory) that should be skipped when doing instrumentation. |
+| istanbulFolder | *String* | `./coverage` |  Folder location for Istanbul coverage reports. |
+| istanbulReporter | *Array* | `['html', 'lcov', 'text', 'json']` | [Istanbul coverage reporters][2]  |
+| mocha | *Object* | `{ }` | [Mocha options][3] to merge into existing mocha config. `grep` and `invert` are useful for skipping certain tests under coverage using tags in the test descriptions.|
+| onServerReady[<sup>*</sup>][14] | *Function* |   | Hook run *after* server is launched, *before* the tests execute. Useful if you need to use the Oraclize bridge or have setup scripts which rely on the server's availability. [More...][23] |
+| onCompileComplete[<sup>*</sup>][14] | *Function* |  | Hook run *after* compilation completes, *before* tests are run. Useful if you have secondary compilation steps or need to modify built artifacts. [More...][23]|
+| onTestsComplete[<sup>*</sup>][14] | *Function* |  | Hook run *after* the tests complete, *before* Istanbul reports are generated. [More...][23]|
+| onIstanbulComplete[<sup>*</sup>][14] | *Function* |  | Hook run *after* the Istanbul reports are generated, *before* the ganache server is shut down. Useful if you need to clean resources up. [More...][23]|
+
+[<sup>*</sup> Advanced use][14]
+
+## Usage
+
++ Coverage runs tests a little more slowly.
++ Coverage uses the Hardhat network by default.
++ Coverage [distorts gas consumption][13]. Tests that check exact gas consumption should be [skipped][24].
```

Comment:
Add note here about increasing your default account balances if you see errors like 
```
expected 'sender doesn\'t have enough funds to send tx. The upfront cost is: 72057636037927928000000000 and the sender\'s account only has: 9999999999999857947383' to include 'revert'
```